### PR TITLE
add precompiled 2.11.0

### PR DESCRIPTION
**Pull request by xuwei-k**
## what this changes

Adds Scala 2.11.0 to the precompiled.
## expectations

The benefit is that the projects using 2.11.0 would boot up faster.

### CODE REVIEW

File: project/Sbt.scala [hunk 1]
```diff
@@ -137,7 +138,7 @@ object Sbt extends Build
 		// Strictly for bringing implicits and aliases from subsystems into the top-level sbt namespace through a single package object
 		//  technically, we need a dependency on all of mainSub's dependencies, but we don't do that since this is strictly an integration project
 		//  with the sole purpose of providing certain identifiers without qualification (with a package object)
-	lazy val sbtSub = baseProject(sbtPath, "sbt") dependsOn(mainSub, compileInterfaceSub, precompiled282, precompiled292, precompiled293, scriptedSbtSub % "test->test") settings(sbtSettings : _*)
+	lazy val sbtSub = baseProject(sbtPath, "sbt") dependsOn(mainSub, compileInterfaceSub, precompiled282, precompiled292, precompiled293, precompiled211, scriptedSbtSub % "test->test") settings(sbtSettings : _*)
```
Comment (jsuereth) [inline project/Sbt.scala]: The name you have here does not match the name above.

**Comment by jsuereth** [comment]
Thanks for pushing this change, but the build failures look legitimate (once you get past the first one).

**Comment by jsuereth** [comment]
Also, because we have a non-empty commit message policy, we'd like some free-verse poetry or a haiku about the weather as you wrote the commit instead.   @eed3si9n can help if needed, he's a master poet.

File: project/Sbt.scala (hunk 1 above)
Comment (eed3si9n) [inline project/Sbt.scala]: In other words a typo `s/precompiled211/precompiled2110/`.

Comment (xuwei-k) [inline project/Sbt.scala]: Oops! I just fixed

**Comment by eed3si9n** [comment]
@xuwei-k did provide a commit message. He just didn't write a description to the pull req, so I made one up as above. Please correct it if there's an issue.
Today's poetry is 春眠不觉晓，处处闻啼鸟。which roughly translates to when you work from home, you get to sleep in late during spring time while listening to the bird chirp.

**Comment by jsuereth** [comment]
Poetry looks good.  Iambic pentameter is only required for native English speakers.  Once travis reports success, good to merge

**Comment by eed3si9n** [comment]
Using sbt 0.13.1 launcher I'm getting:

```
$ sbt 
Getting org.scala-sbt sbt 0.13.5-SNAPSHOT ...
downloading http://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt.ivy/ivy/2.4.0-sbt-d6fca11d63402c92e4167cdf2da91a660d043392/jars/ivy.jar ...
    [SUCCESSFUL ] org.scala-sbt.ivy#ivy;2.4.0-sbt-d6fca11d63402c92e4167cdf2da91a660d043392!ivy.jar (1305ms)
:: retrieving :: org.scala-sbt#boot-app
    confs: [default]
    45 artifacts copied, 0 already retrieved (13677kB/50ms)
Getting Scala 2.11.0 (for sbt)...
:: retrieving :: org.scala-sbt#boot-scala
    confs: [default]
    6 artifacts copied, 0 already retrieved (24354kB/27ms)
java.lang.NoSuchMethodError: scala.collection.immutable.$colon$colon.hd$1()Ljava/lang/Object;
    at sbt.Load$.loadAll(Load.scala:307)
    at sbt.Load$.loadURI(Load.scala:264)
    at sbt.Load$.load(Load.scala:260)
    at sbt.Load$.load(Load.scala:251)
    at sbt.Load$.apply(Load.scala:134)
    at sbt.GlobalPlugin$.build(GlobalPlugin.scala:33)
    at sbt.GlobalPlugin$.load(GlobalPlugin.scala:39)
    at sbt.Load$.loadGlobal(Load.scala:83)
    at sbt.Load$.defaultWithGlobal(Load.scala:64)
    at sbt.Load$.defaultLoad(Load.scala:35)
    at sbt.BuiltinCommands$.doLoadProject(Main.scala:473)
    at sbt.BuiltinCommands$$anonfun$loadProjectImpl$2.apply(Main.scala:467)
    at sbt.BuiltinCommands$$anonfun$loadProjectImpl$2.apply(Main.scala:467)
    at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:60)
    at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:60)
    at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:62)
    at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:62)
    at sbt.Command$.process(Command.scala:95)
    at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:100)
    at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:100)
    at sbt.State$$anon$1.process(State.scala:179)
    at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:100)
    at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:100)
    at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
    at sbt.MainLoop$.next(MainLoop.scala:100)
    at sbt.MainLoop$.run(MainLoop.scala:93)
    at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:71)
    at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:66)
    at sbt.Using.apply(Using.scala:25)
    at sbt.MainLoop$.runWithNewLog(MainLoop.scala:66)
    at sbt.MainLoop$.runAndClearLast(MainLoop.scala:49)
    at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:33)
    at sbt.MainLoop$.runLogged(MainLoop.scala:25)
    at sbt.StandardMain$.runManaged(Main.scala:57)
    at sbt.xMain.run(Main.scala:29)
    at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:57)
    at xsbt.boot.Launch$.withContextLoader(Launch.scala:77)
    at xsbt.boot.Launch$.run(Launch.scala:57)
    at xsbt.boot.Launch$$anonfun$explicit$1.apply(Launch.scala:45)
    at xsbt.boot.Launch$.launch(Launch.scala:65)
    at xsbt.boot.Launch$.apply(Launch.scala:16)
    at xsbt.boot.Boot$.runImpl(Boot.scala:32)
    at xsbt.boot.Boot$.main(Boot.scala:21)
    at xsbt.boot.Boot.main(Boot.scala)
[error] java.lang.NoSuchMethodError: scala.collection.immutable.$colon$colon.hd$1()Ljava/lang/Object;
[error] Use 'last' for the full log.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?
```

Could either of you reproduce this?

**Comment by jsuereth** [comment]
Just reproduced this locally.  Appears 0.13.1 and previous launcher had erroneous "auto" scala-version detection and is labelling sbt as 2.11.  Will have to back out this change for the 0.13 series at a minimum.

**Pull request merged**